### PR TITLE
Please refer to name of branch

### PR DESCRIPTION
**Pull request by xlexi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -85,6 +85,9 @@ app.use(async function (ctx, next) {
   ctx.query = parseQuery(query)
 
   ctx.inet = ctx.request.req.headers['x-forwarded-for'] || ctx.request.ip
+  if (Object.isArray(ctx.inet)) {
+    [ctx.inet] = ctx.inet
```
Comment (trezy) [inline index.js]: What on earth is this doing?

Comment (xlexi) [reply index.js]: Webserver may provide multiple x-forwarded-for headers, I only want the one first one. However if it's not being a proxy it won't be an array

Comment (GeekyDeaks) [reply index.js]: that's xlex's favourite thing @trezy :D array deconstruction - he's getting the first IP if there are more than one

Comment (trezy) [reply index.js]: Ooooooh... I knew it was an array deconstruction, but I was confused by the deconstructor assignment of `[ctx.inet]` I never thought about assigning the deconstructed datum directly to an object property. Very clever. 🤔

Comment (xlexi) [reply index.js]: :D

**Pull request merged**